pallets/runtime/src/multisig.rs

```diff
@@ -640,6 +649,16 @@ impl<T: Trait> Module<T> {
             }
         }?;
 
+        if let Signatory::AccountKey(key) = signer {
```

Comment:
Is it safe to simply skip the `Identity` case? Would it be safer to register identities with their multisigs as well to guarantee that they are not linked to multiple multisigs?

Reply:
We are not imposing any restrictions on identities. A single identity is allowed to be a signer of multiple multisigs and therefore it's safe to ignore.